rush.json

```diff
@@ -346,6 +346,19 @@
             "versionPolicyName": "sdk",
             "shouldPublish": true
         },
+        {
+            "packageName": "@gooddata/react-app-template",
+            "projectFolder": "tools/react-app-template",
+            "reviewCategory": "tools",
+            "shouldPublish": false
+        },
+        {
+            "packageName": "@gooddata/app-toolkit",
+            "projectFolder": "tools/app-toolkit",
+            "reviewCategory": "tools",
+            "versionPolicyName": "sdk",
+            "shouldPublish": true
```

Comment:
Not sure we should publish yet. At least the compliance should see the licences.